internal/web/routes.go

```diff
@@ -85,6 +87,14 @@ func CreateServer(hostService HostService, content fs.FS, config Config) *http.S
 		hostService: hostService,
 	}
 
+	certs, err := cli.ReadWebCertificates()
```

Comment:
This is not good design. Calling cli package makes testing harder.  Instead `tls.Config` should be passed in `main.go`.